MinecraftClient/Mapping/CubeFromWorld.cs

```diff
@@ -0,0 +1,184 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace MinecraftClient.Mapping
+{
+    /// <summary>
+    /// A row of blocks that will be mined
+    /// </summary>
+    public class Row
+    {
+        private List<Location> blocksInRow;
+
+        public List<Location> BlocksToMine
+        {
+            get { return blocksInRow; }
+        }
+
+        /// <summary>
+        /// Initialize a row of blocks
+        /// </summary>
+        /// <param name="bIL"> Enter a list of blocks </param>
+        public Row(List<Location> bIL = null)
+        {
+            blocksInRow = bIL ?? new List<Location>();
+        }
```

Comment:
Maybe use full name here for better readability?
```suggestion
        public Row(List<Location> blocksInRow = null)
        {
            this.blocksInRow = blocksInRow ?? new List<Location>();
        }
```